lib/cjs/vestaAPI.js

```diff
@@ -49,6 +49,11 @@ class Vesta {
     }
     postMessage(subscriptionId, postMessage) {
         return __awaiter(this, void 0, void 0, function* () {
+            if (typeof postMessage === 'string') {
```

Comment:
In theory, the check for invalid characters could wrap this entire function instead of using the double conditional I have here. Let me know if that's the way you'd prefer.

Reply:
I think this is fine. I don't have a strong preference.